.travis.yml

```diff
@@ -25,6 +25,7 @@ php:
   - 7.1
   - 7.2
   - 7.3
```

Comment:
I think we can remove these two, they are useless

Reply:
I'm not sure, since PHP introduces BC breaks on each version, we need to know which one is affected, sometimes. I'm afraid of having something working on 7.2, 7.4 and not working on 7.3 due to a bug or a BC breaks :/ 
Since our QA team is not testing all PHP versions, we should let Travis do it.